### DISCUSSION TITLE
Have a board to open source tasks?

### DISCUSSION THREAD

**Discussion by juninhopo** [zed-industries/zed#6999] [Feature Requests]
I was making a PR for the Vim commands that work in the folders, but then I ended up finding a PR that was already doing this.

We manage to organize what we want to do and what you need to do, so I can get a "task" that you have on your board.

Could this be interesting for you?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
